index.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 const _ = require('lodash');
+const bugsnag = require("bugsnag");
+const uuid = require("node-uuid-v4");
```

Comment:
should use this npm instead https://www.npmjs.com/package/uuid and use version 1